### PR TITLE
Basic config

### PR DESCRIPTION
**Pull request by boris-arkenaar**
This will fix Signalen/frontend#2, allowing us to configure the logo, colors, and some texts on the website.

* Configuration for these settings can be injected through a JSON file during deployment of the docker container. A `start.sh` has been added to facilitate injecting these settings into the `index.html`.
* For the development server an optional `config.json` in the root of the project will be used to inject into the `index.html`. This is done in the Webpack configuration.
* The existing configuration service is being used. All config from the injected JSON file will be added to the configuration in this service.
* An extra property `all` is added to the configuration service to retrieve all config as an object (except for all the endpoints provided through getters).

- [x] Configuration is possible through a JSON file
- [x] Configuration is injected into index.html (dev server)
- [x] Configuration is injected into index.html (docker deploy)
- [x] Logo is configurable
- [x] Theming variables, as accepted by [Amsterdam/amsterdam-styled-components](https://github.com/Amsterdam/amsterdam-styled-components), is configurable
- [x] Site's footer text is configurable

Before opening a pull request, please ensure:

- [x] double-check your branch is based on `develop` and targets `develop` 
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Internal code generators and templates are updated (if necessary)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/components/SiteHeader/index.js [hunk 1]
```diff
@@ -269,11 +267,19 @@ const MenuItems = ({ onLogOut, showItems }) => {
...
 };
 
+const Logo =
+  configuration.logoUrl &&
+  (props => (
+    <a href={configuration.links.home}>
+      <img alt="Logo" src={configuration.logoUrl} style={{ height: configuration.logoHeight }} {...props} />
+    </a>
+  ));
+
```
Comment (janjaap) [inline src/components/SiteHeader/index.js:277]: Wouldn't use inline styles here. Instead, make it a styled component.

File: environment.conf.json [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+  "logoHeight": 0,
+  "language": {
+    "footer1": "Bel het Gemeentelijk informatienummer: 14 020",
+    "footer2": "op werkdagen van 08.00 tot 18.00 uur."
+  },
+  "links": {
+    "home": "https://www.amsterdam.nl",
+    "privacy": "https://www.amsterdam.nl/privacy/"
+  },
+  "styledComponents": {},
```
Comment (janjaap) [inline environment.conf.json:12]: These props seem to be missing in the other config files

File: src/components/SiteHeader/index.js (hunk 1 above)
Comment (boris-arkenaar) [reply src/components/SiteHeader/index.js]: Separated the component and added unit tests.

**Comment by boris-arkenaar** [comment]
So, we're waiting for the infrastructure of Amsterdam to be ready for this change, before we can merge this.

**Comment by bartjkdp** [comment]
This change seems te break the review app. Maybe we can inject some defaults into the Netlify. build as well:

image

**Pull request merged**